### ISSUE TITLE
Add spinner for service create

### ISSUE THREAD

**Issue by cdrage** [redhat-developer/odo#1063]
Quick issue, we should add spinners / logging for service create

**Comment by kadel** [comment]
I'm not sure if it makes sense to have a spinner for `service create`. It is not waiting for service to be created.

```
▶ odo service create mysql-persistent
Service 'mysql-persistent' was created.
Progress of the provisioning will not be reported and might take a long time.
You can see the current status by executing 'odo service list'
```



```
✓ Service 'mysql-persistent' was created.

Progress of the provisioning will not be reported and might take a long time.
You can see the current status by executing 'odo service list'
```

**Comment by geoand** [comment]
I had a PR that added the wait flag but I think we closed it because it was very out of date.
That said I can easily create a new one and a spinner to it.
What do you think?

**Comment by jorgemoralespou** [comment]
I think it makes sense, but @kadel is the master :-D

**Comment by geoand** [comment]
Just give me the word @kadel and I will get on it 😁

**Comment by geoand** [comment]
Actually it seems like I had added the `--wait` flag for service linking, but not for service create. I can do that in the similar fashion if we think it makes sense

**Comment by kadel** [comment]
`odo service create` shouldn't wait by default.
But adding `--wait` flag makes sense.

**Comment by geoand** [comment]
I'll take this since it's related to #1144

**Comment by kadel** [comment]
closed in #1321
